plugins/bitrot/rest.go

```diff
@@ -141,7 +137,8 @@ func bitrotDisableHandler(w http.ResponseWriter, r *http.Request) {
 	volinfo.Options[keyFeaturesScrub] = "false"
 
 	if err := txn.Ctx.Set("volinfo", volinfo); err != nil {
-		logger.WithError(err).Error("failed to set volinfo in transaction context")
+		logger.WithError(err).WithField("volname",
```

Comment:
@Madhu-1 We have decided a generic message to be logged when setting a key in transaction context fails.https://github.com/gluster/glusterd2/blob/master/plugins/device/rest.go#L85.
It becomes easier to debug. Though this also works but providing the key in the WithField is helpful in debugging.

Reply:
I guess you have asked for an explanation for the same in other PRs as well.